distributed-tracing/docker-compose.yml

```diff
@@ -4,7 +4,8 @@ x-otel-common:
   OTEL_TRACES_SAMPLER: parentbased_always_on
   OTEL_TRACES_EXPORTER: otlp
   OTEL_EXPORTER_OTLP_TRACES_PROTOCOL: grpc
-  OTEL_EXPORTER_OTLP_ENDPOINT: host.docker.internal:4317
+  OTEL_EXPORTER_OTLP_ENDPOINT: stagingapp.signoz.io:4317
```

Comment:
@prashant-shahi should not be pointing to `staging`?

Reply:
replacing it with host signoz endpoint